src/components/atoms/authors/authors.module.scss

```diff
@@ -18,19 +19,97 @@
 
   display: inline;
 
-  &::after {
+  &::before {
     content: ", ";
   }
 
-  &:last-child::after {
-    content: "";
+  &:first-of-type {
+    &::before {
+      content: "";
+    }
+  }
+
+  .authors--collapsible & {
+    &:nth-of-type(1n+3) {
+      display: none;
+    }
+
+    &:last-of-type {
+      display: inline;
+
+      &::before {
+        content: " ... ";
+      }
+    }
+
+    @media only all and (min-width: settings.$breakpoint-medium) {
+      &:nth-of-type(1n+3) {
+        display: inline;
+      }
+
+      &:last-of-type {
+        &::before {
+          content: ", ";
+        }
+      }
+
+      &:nth-of-type(1n+11) {
+        display: none;
+      }
+    }
+  }
+
+  .authors-list--expanded & { /* stylelint-disable-line no-descending-specificity */
+    &:nth-of-type(1n+3) {
+      display: inline;
+    }
+
+    &:last-of-type {
+      display: inline;
+
+      &::before {
+        content: ", ";
+      }
+    }
   }
 }
 
 .authors-list__expansion {
-  @include typography.font-size-and-vertical-height(16, 24);
-  @include typography.expansion-text();
+  display: none;
+
+  .authors--collapsible & {
+    @include typography.font-size-and-vertical-height(16, 24);
+    @include typography.expansion-text();
+
+    cursor: pointer;
+    font-weight: 600;
+
+    .authors-list__expansion-count {
+      display: none;
+
+      @media only all and (max-width: settings.$breakpoint-medium) {
+        &.authors-list__expansion-count-3 {
+          display: inline;
+        }
+      }
 
-  cursor: pointer;
-  font-weight: 600;
+      @media only all and (min-width: settings.$breakpoint-medium) {
+        &.authors-list__expansion-count-10 {
```

Comment:
Just gone through this again. Why do we have magic numbers on classes like here and on line 91, 104 and 110?